src/Scene/NodeProcess.js

```diff
@@ -205,18 +213,12 @@ NodeProcess.prototype.hideNodeChildren = function(node) {
  */
 function findAncestorWithValidTextureForLayer(node, layerType, layer) {
     var parent = node.parent;
-    if (parent && parent.material && parent.material.getLayerTextureOffset) {
-        var slot = layerType == l_ELEVATION ? 0 : parent.material.getLayerTextureOffset(layer.id);
-        if (slot < 0) {
-            return null;
+    if (parent && parent.material && parent.material.getLayerLevel) {
+        var level = parent.material.getLayerLevel(layerType, layer);
+        if (level >= 0) {
```

Comment:
I'd prefer to keep the other order `0 <= level`
I like to only use `<` and `<=` and never use `>`. Of course in single statement it doesn't matter, but in more complex case :
```js
if (a >= -2 && a <= 10)
if (-2 <= a && a <= 10)
```
I think the 2nd version is easier to read.